roles/installer/ospd/overcloud/scale/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+- name: Set scaler variables
+  set_fact:
+    scale_service={{ item.key }}
+    scale_amount={{ item.value }}
+  when: (item.key == "ceph-storage") or
```

Comment:
why do we need a "when"?
When you invoke the role, you call it with the param you want to scale, please look at: http://docs.ansible.com/ansible/playbooks_roles.html#roles